### DISCUSSION TITLE
Multi-language support

### DISCUSSION THREAD

**Discussion by mizaki** [comictagger/comictagger#397] [Ideas]
Has there been much/any request for supporting multiple languages?

**Comment by lordwelch** [comment]
Not that I have seen

**Comment by mizaki** [comment]
Do you think it's a chicken and egg situation? There is `gettext` for python (as you probably already know) and plenty of translation websites that host free for open source. I think the only problem might be different lengths for words and fitting in the buttons etc.

Wonder if it's worth the effort...?

**Reply by lordwelch** [reply]
Qt already has a system for translations (the tr function that they stick on every object they can) and from what I've seen most CLIs are just left in english as it is mostly just for automation...
Mainly I've just not seen any requests for it to be translated.
I didn't know what the `tr` function did at the time (I thought it was a string conversion leftover from python2) so I removed them in e10f7dd7a796ed5c8dc4e6422ef1fc1248d02a88

**Reply by mizaki** [reply]
I've noticed from issues and in a discussion comics and sources for other languages so I was wondering there. Also, I imagine if you use it, you can understand English well enough to but if you don't you won't use it. I know with some software that wasn't in English, I had no idea how to ask for one.

I'm wondering on how well it works with a screen reader too?

I'm also wondering if a new UI (and conversion to Qt6) then too...
